contracts/ExpectedRate.sol

```diff
@@ -76,6 +97,13 @@ contract ExpectedRate is Withdrawable, ExpectedRateInterface, Utils2 {
         return (expectedRate, slippageRate);
     }
 
+    function checkKncArbitrageRate(uint currentKncToEthRate) public view returns(bool) {
+        uint converseRate; uint slippage;
+	(converseRate, slippage) = getExpectedRate(ETH_TOKEN_ADDRESS,knc,10**18,true);
+        require(converseRate <= MAX_RATE && currentKncToEthRate <= MAX_RATE);
+        return ((converseRate * currentKncToEthRate) > (PRECISION ** 2));
+    }
```

Comment:
in fee burner we have this line.
require(kyberEthKncRate * kyberKncEthRate > PRECISION ** 2 / 2);

why should the line above be added?